src/BlockTypes/MiniCart.php

```diff
@@ -306,11 +306,19 @@ protected function get_markup( $attributes ) {
 			</div>';
 		}
 
-		$part                   = 'mini-cart';
-		$template_part          = gutenberg_get_block_template( get_stylesheet() . '//' . $part, 'wp_template_part' );
 		$template_part_contents = '';
-		if ( $template_part && ! empty( $template_part->content ) ) {
-			$template_part_contents = do_blocks( $template_part->content );
+		if ( function_exists( 'gutenberg_get_block_template' ) ) {
+			$part          = 'mini-cart';
+			$template_part = gutenberg_get_block_template( get_stylesheet() . '//' . $part, 'wp_template_part' );
```

Comment:
Because we only use `$part` one time, I think we should use the string directly.
```suggestion
			$template_part = gutenberg_get_block_template( get_stylesheet() . '//mini-cart', 'wp_template_part' );
```

Reply:
Makes sense!